Client/Frontend/Widgets/Toolbar.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 import Snap
 
-public let StatusBarHeight: CGFloat = 20 // TODO: Can't assume this is correct. Status bar height is dynamic.
+public let StatusBarHeight: CGFloat = UIApplication.sharedApplication().statusBarFrame.size.height
```

Comment:
Calculating status bar height dynamically instead of setting it to a value(20).
Reference http://stackoverflow.com/questions/12991935/how-to-programmatically-get-ios-status-bar-height